pkg/logging/logging.go

```diff
@@ -32,31 +33,9 @@ const (
 	MutationApplied       = "mutation_applied"
 	Mutator               = "mutator"
 	DebugLevel            = 2 // r.log.Debug(foo) == r.log.V(logging.DebugLevel).Info(foo)
+	StatsEntries          = "stats_entries"
```

Comment:
just `stats`? Not sure what extra information "entries" adds. Maybe `execution_stats`?

Reply:
I think `execution_stats` works for now. If there are other types of stats in the future being logged here, we may want to revisit this but since we control the call site it shouldn't be a problem.